front/app/api/projects_mini/useProjectsMini.ts

```diff
@@ -7,20 +7,18 @@ import { getPageNumberFromUrl } from 'utils/paginationUtils';
 import miniProjectsKeys from './keys';
 import { MiniProjects, MiniProjectsKeys, QueryParameters } from './types';
 
-const fetchProjectsWithActiveParticipatoryPhase = (
-  queryParameters: QueryParameters
-) =>
+const fetchProjectsMini = (queryParameters: QueryParameters) =>
   fetcher<MiniProjects>({
-    path: '/projects/with_active_participatory_phase',
+    path: `/projects/${queryParameters.endpoint}`,
     action: 'get',
     queryParams: {
       'page[size]': queryParameters['page[size]'] ?? 6,
       'page[number]': queryParameters['page[number]'] ?? 1,
     },
```

Comment:
The `endpoint` doesn't really seem like a queryParameter though as it's used now. Do you think there is a way we could have a single endpoint (E.g. "/projects/mini") and then have a param in the query params to pass `with_active_participatory_phase` or `for_followed_item` instead? Maybe some sort of `filter_type` param or something? :thinking:

Reply:
Hmm I think the backend structure is fine like this, but I agree that calling the endpoint a query parameter is not accurate. I will rename the variable and type to just `parameters` if that's ok?

Reply:
Yeah `parameters` sounds good then! :) 

Do you think we'll be adding more `endpoint` types going forwards? It seems to me like it could make sense to have a single BE endpoint rather than create multiple separate ones for each endpoint type (if it's used for filtering the projects list)? :thinking: Though indeed I don't have a lot of experience with our BE code and this feature specifically, so if it's cleaner to have them separated out then sounds good! :+1:

Reply:
I will ask Simon 👍

Reply:
Gotcha! Yeah that's very clear - thanks for the thorough explanation!! :+1: